### DISCUSSION TITLE
How to switch between multiple sources of video

### DISCUSSION THREAD

**Discussion by pktiuk** [RosettaDrone/rosettadrone#147] [Q&A] [answered]
I have a drone with attached camera running in two modes. Regular one and IR.    
How can I switch between them?

**Comment by kripper** [comment]
Check the SDK documentation.

**Comment by kripper** [comment]
Also check the DJI SDK Examples and maybe also the DJI Video Sample App.

**Reply by pktiuk** [reply]
Sorry, but I don't understand. I was asking where in the GUI of the Rosetta Drone is a button for switching between cameras. Regular DJI app has a button with word IR/Zoom/Wide for switching between cameras
![image](https://user-images.githubusercontent.com/45544416/230180436-8db26dc8-49aa-422b-bd09-86f5e0eeca8b.png)


Does this mean, that Rosetta Drone doesn't have this functionality?

**Comment by kripper** [comment]
Sorry  I have no information. My model doesn't support it.

**Comment by The1only** [comment, marked as answer]
We did not add that widget, however I do remember considering it.
It is quite easy to add it you’re self thow. 
All thows widgets comes from the DJI UI sdk ready to be used.

**Reply by pktiuk** [reply]
Thank you for precise answer
